srs_extension/test/runTest.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+
+const { runTests } = require('vscode-test');
+
+async function main() {
+    try {
+        // The folder containing the Extension Manifest package.json
+        // Passed to `--extensionDevelopmentPath`
+        const extensionDevelopmentPath = path.resolve(__dirname, '../');
+
+        // The path to the extension test script
+        // Passed to --extensionTestsPath
+        const extensionTestsPath = path.resolve(__dirname, './suite/index');
+
+        // Download VS Code, unzip it and run the integration test
+        await runTests({ extensionDevelopmentPath, extensionTestsPath });
+    } catch (err) {
+        console.error('Failed to run tests');
+        process.exit(1);
+    }
+}
```

Comment:
We could run this as part of the gate. Consider.

Reply:
Giving this a try (it seems to run from a command prompt on my machine so we'll see if it works in the gate too)